### DISCUSSION TITLE
Reinforcement

### DISCUSSION THREAD

**Discussion by expertegorov** [IfcOpenShell/IfcOpenShell#3884] [Q&A] [answered]
Greetings. Perhaps someone knows why this happens.
BlenderBIM:
![BlenderBIM](https://github.com/IfcOpenShell/IfcOpenShell/assets/123471482/215cab30-0dc4-4023-845d-99e614bb966b)
BIMcollab:
![BIMcollab](https://github.com/IfcOpenShell/IfcOpenShell/assets/123471482/7623513f-0bb7-4083-9af2-5a265eb835cb)
IFC file:
[01.zip](https://github.com/IfcOpenShell/IfcOpenShell/files/12891236/01.zip)

**Comment by Moult** [comment, marked as answer]
This is a bug in BBIM due to an incorrect handling of mapped representations most likely. I'll tackle it :)

**Reply by Moult** [reply]
Works now!
![2023-10-13-233958_933x595_scrot](https://github.com/IfcOpenShell/IfcOpenShell/assets/88302/8b91794e-0922-4211-81df-6d5e8f2776f9)
